model/workflow/registry.yaml

```diff
@@ -0,0 +1,353 @@
+groups:
+  - id: registry.workflow
+    type: attribute_group
+    brief: >
+      Workflows seek to describe the execution of one ore more tasks to achieve an end result.
+
+      Workflows can be used to describe cicd, deployments, generic jobs or even cron jobs.
```

Comment:
A consumer of SemConv for CI/CD might not know to look in `workflow` for CI/CD conventions.

What can we do to make this clearer?

Can we improve the docs/cicd/README.md or docs/cicd/cicd-spans and link to the workflows page?